### DISCUSSION TITLE
When I re export Mixamo's FBX file from Blender,  it unable to apply correctly to VRM.

### DISCUSSION THREAD

**Discussion by wmlgl** [pixiv/three-vrm#1328] [Help]
I used loadMixamoAnimation function to load it.

I created an animation clip using Mixamo 's y-bot and Blender, and exported it to FBX, but  it unable to apply correctly to VRM.
I also tried exporting unmodified Mixamo's animated FBX from Blender, but there is the same issue.
The animation was works well with Three.js with the FBX file itself.

It's need to set some export parameters for FBX?

thx.

**Comment by 0b5vr** [comment]
@wmlgl 
Does our example work correctly using the file you have downloaded from Mixamo?
https://pixiv.github.io/three-vrm/packages/three-vrm/examples/humanoidAnimation/index.html

Would you provide a demo using sandbox services to describe your issue? You can use [Glitch](https://glitch.com/), [jsfiddle](https://jsfiddle.net/), [CodeSandbox](https://codesandbox.io/), etc.

**Reply by wmlgl** [reply]
Not work correctly on the example. https://psychedelic-false-cowbell.glitch.me/test.html

And this is fbx itself. https://psychedelic-false-cowbell.glitch.me/index.html

I compared the Fbx data loaded from Three. js and found that it has a different attribute called inheritType, which is related to the transformation.

Basically, the FBX animation exported from Blender does not work correctly. blender3.6.5, 2.93, 2.80
